src/core/storage/preferences/preferencesStorage.ts

```diff
@@ -1,46 +0,0 @@
-import { Preferences } from "@capacitor/preferences";
-import {
-  RemoveOptions,
-  SetOptions,
-} from "@capacitor/preferences/dist/esm/definitions";
```

Comment:
This dependency can go from the package.json and lock now

Reply:
@iFergal Will I remove it from package.json and package-lock.json?

Reply:
just `npm uninstall @capacitor/preferences` and commit the changes